feat: new importmap tool

tatang26 opened the pull request:
Introducing the new importmap tool

paganotoni commented:
@tatang26 I took a brief look at the code and these are my finding so far:
- We should add tests
- Could we make sure we are using gofumpt locally?
- Some of the functions need to have some comments.
- Some errors are not being handled.


I really appreciate the progress in  here!

tatang26 commented:
Thank you @paganotoni , these are the changes I added from your feedback:

- I added tests to cover all import-map commands
- I added a tool usage that will be printed when exec `go tool importmap` 
- All commands print the execution results whatever the result is
- Almost all errors are handled, those that are ignored won't generate unexpected executions, they only will return empty values that are managed later
- I formatted all files using `gofumpt`

Please let me know if there's anything else you'd like me to tweak!

paganotoni reviewed (approved):
Looks good @tatang26 !
